### ISSUE TITLE
[Adaptive Cards - Blog]: 'Adaptive Cards Team' link under Blog screen looks like a plain text. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8540]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 23H2 (OS Build 25381.1)
URL: https://adaptivecards.io/blog/
Browser: Edge Dev Version 115.0.1880.3 (Official build) dev (64-bit)

**Repro Steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to 'Blog' tab item and press enter key to activate it.
3. Navigate to the 'Adaptive Cards Team' link and observe the issue.

**Actual Result:**
 'Adaptive Cards Team' link looks like plain text and while hovering on it there is no underline.

**Expected Result:**
'Adaptive Cards Team' link should be display as a link instead of plain text. It should be chevron or underline so that user can easily identify that it is a link.

**Note:**
This issue is repro throughout the Blog screen.

**User Impact:**
It will impact the low vision user as they might get confused and not able to differentiate between normal and link text.

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

![Usable - 'Adaptive Cards Team' link under Blog screen looks like a plain text](https://github.com/microsoft/AdaptiveCards/assets/95913748/667cc02d-9f7f-4342-b543-254280379fb5)

### Card JSON

```JSON
NA
```
